validation_tests/cinema/clean.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/bash
```

Comment:
/usr/bin/bash doesn't exist on some (my) systems. I think /bin/bash is a more reliable standard.